### PR TITLE
Configure gsoci.azurecr.io as the registry to use by default

### PR DESCRIPTION
**Pull request by marians**
Towards

- https://github.com/giantswarm/roadmap/issues/3017

This PR replaces any occurrence of `docker.io` in `values.yaml` files with `gsoci.azurecr.io`,
to make the new ACR registry the default one.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -319,31 +319,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-### Removed
-
-- Remove creation of cilium app config.
-
-## [13.0.0-alpha2] - 2022-07-27
-
-### Changed
-
-- Bump k8scc to fix apiserver's flags and make metrics-server to work.
+- Bump `k8scc` to enable authn and authz on `scheduler` and `controller-manager`.-
```
Comment (fiunchinho) [inline CHANGELOG.md:322]: Something went wrong here @marians

Comment (marians) [reply CHANGELOG.md]: Done

File: helm/aws-operator/values.yaml [hunk 2]
```diff
@@ -66,7 +66,8 @@ tenant:
     clusterDomain: cluster.local
     networkSetup:
       docker:
-        image: giantswarm/k8s-setup-network-environment:1f4ffc52095ac368847ce3428ea99b257003d9b9
+        image:
+          giantswarm/k8s-setup-network-environment:1f4ffc52095ac368847ce3428ea99b257003d9b9
```
Comment (fiunchinho) [inline helm/aws-operator/values.yaml:70]:
```suggestion
        image: giantswarm/k8s-setup-network-environment:1f4ffc52095ac368847ce3428ea99b257003d9b9
```

Comment (marians) [reply helm/aws-operator/values.yaml:70]: FYI `ruamel.yaml` (the python module) did that automatically, obviously to prevent overly long lines

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**